library/globals.inc.php

```diff
@@ -1615,7 +1615,12 @@
       '1',
       xl('New form validation')
      ),
-
+      'safe_key' => array(
+          xl('Safe key'),
+          'text',                           // data type
+          '',                               // default
+          xl('Safe key for security database password')
+      ),
```

Comment:
Also recommend adding a global above this that controls whether Zend even does a database lookup:
Maybe 'Allow connection to other databases'